sycl/test-e2e/Reduction/reduction_deterministic.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <cstdio>
+#include <cstdlib>
+#include <random>
+
+#include <sycl/detail/core.hpp>
+#include <sycl/ext/oneapi/experimental/reduction_properties.hpp>
+
+namespace syclex = sycl::ext::oneapi::experimental;
+
+float sum(sycl::queue q, float *input, size_t N) {
+
+  float result = 0;
```

Comment:
Highly subjective, but I'd rather see

```
  sycl::buffer<float, 1> buf{1}; // Or even sycl::buffer<float, 0>
  q.submit();
  return sycl::host_accessor{buf}[0];
```

Same in another file.

Reply:
Thanks, this does look nicer.  Fixed in 56c149cbd8d1b326a166416eef7ceb06cf55924a.